static/js/modules/election-lookup.js

```diff
@@ -222,6 +223,7 @@ ElectionLookup.prototype.search = function(e, opts) {
     self.serialized = serialized;
     if (opts.pushState) {
       window.history.pushState(serialized, null, URI('').query(serialized).toString());
+      analytics.pageView();
```

Comment:
No, but submitting the election search form on the home page triggers a new page load, which sends the event to GA.